scss/civicrm/contact/pages/_add-to-group.scss

```diff
@@ -1,19 +1,20 @@
-.CRM_Contact_Form_Task_AddToGroup {
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+
+.CRM_Contact_Form_Task_AddToGroup,
+.CRM_Event_Form_Task_AddToGroup {
   .crm-contact-task-addtogroup-form-block {
     > .form-layout {
       > tbody > tr > td {
-        padding: 20px;
+        padding: 20px 0 0 20px;
       }
 
       .label {
         padding-left: 0 !important;
       }
     }
-  }
 
-  #id_existing_group {
-    > td {
-      padding: 20px;
+    .crm-submit-buttons {
+      margin-top: 20px;
```

Comment:
shouldn't the "submit buttons" block always have the same margin in every page?

Reply:
Yes, Thogh on this page, We need to have gap between forms and buttons so, this one is due to that. see below screenshot
![image](https://user-images.githubusercontent.com/26058635/37700861-08a1919a-2d13-11e8-84b2-06b15e27addb.png)